tests/fixture_valid.js

```diff
@@ -232,10 +233,17 @@ module.exports.checkFixture = function checkFixture(fixture, usedShortNames=[])
           if (!mode.channels.includes(fineChannels[ch])) {
             result.errors.push(`Mode '${modeShortName}' contains the fine channel '${ch}' (#${chNumber}) but is missing its coarse channel '${coarseChannelKey}'.`);
           }
-          // the mode must also contain all coarser channel
+          // the mode must also contain all coarser channels
           for (let fineIndex = 0; fineIndex < fineChannelAliases.indexOf(ch); fineIndex++) {
             let coarserChannelKey = fineChannelAliases[fineIndex];
-            if (!mode.channels.includes(coarserChannelKey)) {
+
+            // check if the coarse channel is used directly or as part of a switching channel
+            if (!mode.channels.some(
```

Comment:
This is not readable at all. Please give it an own well-named variable (which makes the if statement intuitively understandable) and do the same for the inner `some`s.